src/pal/inc/pal.h

```diff
@@ -5608,6 +5608,12 @@ PALIMPORT
 DWORD
 PALAPI
 GetCurrentProcessorNumber();
+
+PALIMPORT
+BOOL
+PALAPI
+PAL_HasFunctionality(
```

Comment:
Please add a commented describing the purpose of this function.
What do you think about changing its name to PAL_HasCapability?
funcMask => functionalityMask or capabilityMask?